### DISCUSSION TITLE
Grotesque Guardians Falling Rock Indiactor

### DISCUSSION THREAD

**Discussion by S1rCumference** [runelite/runelite#14917] [Ideas]
Ive been recently doing Grotesque Guardians and have about 100kc its a easy boss but one of the things that occasionally messes me up is my lack of depth perception/calculation of where to stand for the falling rocks in the 2nd phase. It would be super helpful to have the feature like in the Zalcano plugin that shows the tile your standing on and the one falling. Im will to pay at this point to get it made since im tired of being stunned  then stacked out by the orange light

**Comment by S1rCumference** [comment]
Honestly i think having a separate section for all falling obstacles with this feature would be helpful to many. Unsure if its added but i had similar difficulty with the Ohm fight. Im sure there are many other similar instances

**Comment by Bram91** [comment]
This is against the rules outlined here: 
https://secure.runescape.com/m=news/another-message-about-unofficial-clients?oldschool=1
> Anything that automatically indicates where to stand, or not to stand. This applies to only automatic indicators, and not tiles which have been manually marked.

and
> Projectile targets, or target locations, as well as impact locations

**Reply by S1rCumference** [reply]
Thanks didnt know about that. Wonder why the Zalcano one exists then. Maybe there could be a recolor option to the monotone grey shadow then to something brighter. Im sure that colourblind people would appreciate that aswell

**Reply by Enriath** [reply]
The Zalcano one exists because Jagex ok'd OSB while they had it, which is odd; maybe they don't consider Zalcano as a boss or something.

**Comment by pwatts6060** [comment]
You may find the core Camera plugin useful. You can enable/disable a setting so that the vertical angle isn't limited giving you a direct top down view which I find helpful for content like vorkath where there is an aoe projectile to dodge.

**Reply by S1rCumference** [reply]
Thanks I’ll check it out!
